src/test/java/io/github/enkarin/bookcrossing/support/TestDataProvider.java

```diff
@@ -125,4 +125,24 @@ public static UserRegistrationDto buildAlex() {
                 .email("t.test@mail.ru")
                 .build();
     }
+
+    @Nonnull
+    public static UserRegistrationDto buildNonConfirmedPasswordUser() {
+        return UserRegistrationDto.builder()
```

Comment:
You can use the prepareUser() to not write the definition of some fields

Reply:
Sure) Changed it